test/jit/test_freezing.py

```diff
@@ -891,3 +891,47 @@ def test_freeze_module_detach_gradient(self):
         out3 = smod(inp)
         self.assertNotEqual(out1, out2)
         self.assertEqual(out2, out3)
+
+    def test_freeze_module_with_user_preserved_attr(self):
+        class Module(nn.Module):
+            def __init__(self):
+                super(Module, self).__init__()
+                self.a = torch.tensor([1.1])
+                self.b = torch.tensor([2.2])
+
+            def forward(self, x):
+                return self.a + self.b
+
+        m = torch.jit.script(Module())
+        m.eval()
+        fm = torch._C._freeze_module(m._c, ["a"])
+        # Attribute "a" is preserved
+        self.assertTrue(fm.hasattr("a"))
+        self.assertFalse(fm.hasattr("b"))
+
+    def test_freeze_module_with_user_preserved_method(self):
```

Comment:
Can you add a test like:

        class Module(nn.Module):
            def __init__(self):
                super(Module, self).__init__()
                self.a = torch.tensor([1.1])
                self.b = torch.tensor([2.2])

            def forward(self, x):
                self.b[0] += 10
                return self.a + self.b

            @torch.jit.export
            def modify_a(self, x):
                self.a[0] += 10 
                return self. b + self.a

        m = torch.jit.script(Module())
        m.eval()
        fm = torch._C._freeze_module(m._c, ["modify_a"])
        FileCheck().check('prim::GetAttr[name="a"]').run(fm.forward.graph)
        FileCheck().check('prim::GetAttr[name="b"]').run(fm.modify_a.graph)